backend/src/main/java/com/bakdata/conquery/util/search/TrieSearch.java

```diff
@@ -180,10 +179,19 @@ private Stream<T> doGet(String kw) {
 
 
 	private void doPut(String kw, T item) {
+		assertWriteable();
```

Comment:
Ich würde es nicht assert nennen, weil assertions zur Runtime zu noops werden sollten außer man startet mit `-ea`

Reply:
> Ich würde es nicht assert nennen, weil assertions zur Runtime zu noops werden sollten außer man startet mit -ea

Habe es mal in ensureWriteable umbenannt?